includes/definitions/ztt.yaml

```diff
@@ -0,0 +1,17 @@
+os: ZTT
+text: 'ZTT'
+type: power
+icon: ztt
+group: ZTT
+mib dir: ztt
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.49692
+            - .1.3.6.1.4.1.49692.1.
```

Comment:
You don't need line 13 to 17. Line 12 include this lines

Reply:
Thanks